roles/openshift_health_checker/library/ocutil.py

```diff
@@ -0,0 +1,43 @@
+# pylint: disable=missing-docstring
+
+"""Interface to OpenShift oc command"""
+
+import shlex
+import subprocess
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            namespace=dict(type="str", required=True),
+            config_file=dict(type="str", required=True),
+            cmd=dict(type="str", required=True)
+        ),
+    )
+
+    cmd = ['oc',
```

Comment:
I just tried running this with a containerized install and of course... no `oc`.

I'm going to look at how other code handles OpenShift API interaction.